lambda_functions/delivery_status_processor_lambda/delivery_status_processor_lambda.py

```diff
@@ -34,12 +36,56 @@
     logger.warning("Invalid log level specified, defaulting to INFO")
 
 
+# Duplicated in vetext_incoming_forwarder.
+def validate_twilio_event(event):
+    logger.info("validating twilio delivery event")
+
+    try:
+        ssm_client = boto3.client("ssm", "us-gov-west-1")
+        auth_ssm_key = os.getenv("TWILIO_AUTH_TOKEN_SSM_NAME", "")
+        if not auth_ssm_key:
+            logger.error("TWILIO_AUTH_TOKEN_SSM_NAME not set")
+            return False
+
+        response = ssm_client.get_parameter(
+            Name=auth_ssm_key,
+            WithDecryption=True
+        )
+        auth_token = response.get("Parameter").get("Value")
```

Comment:
No SSM information should be looked up during an invocation. This should be done during execution environment setup.